src/java.base/share/classes/sun/util/calendar/AbstractCalendar.java

```diff
@@ -167,29 +167,16 @@ public long getTime(CalendarDate date) {
             }
             // adjust time zone and daylight saving
             int[] offsets = new int[2];
-            if (date.isStandardTime()) {
```

Comment:
This seems a little suspicious considering `isStandardTime()` is always false.

However, at this point, there shouldn't be any behavioral changes probably.

Inlined isStandardTime() here as false, thus removing the code block contained in the _if_